core/collateral/engine.go

```diff
@@ -1898,14 +1898,15 @@ func (e *Engine) getLedgerEntries(ctx context.Context, req *types.TransferReques
 				return nil, err
 			}
 			for _, to = range ret.Balances {
-				lm = &types.LedgerEntry{
-					FromAccount: acc.ToDetails(),
-					ToAccount:   to.Account.ToDetails(),
-					Amount:      parts,
-					Type:        req.Type,
-					Timestamp:   now,
-				}
-				ret.Entries = append(ret.Entries, lm)
+				ret.Entries = append(ret.Entries, &types.LedgerEntry{
+					FromAccount:                 acc.ToDetails(),
+					ToAccount:                   to.Account.ToDetails(),
+					Amount:                      parts,
+					Type:                        req.Type,
+					Timestamp:                   now,
+					SenderPostTransferBalance:   acc.Balance.Clone(),
+					ReceiverPostTransferBalance: num.Sum(to.Account.Balance, parts),
```

Comment:
Kind of minor but it I think it would be nice to be consistent with the terminology, e.g. From/Sender and To/Receiver

Also to this day I can never spell receiver right the first time

Reply:
I dont understand what do you suggest here tbh

Reply:
e.g. maybe rename `SenderPostTransferBalance` to `FromAccountPostTransferBalance`, or maybe just 'FromAccountBalance'.  It's just a suggestion, not a request - it just jars a little bit that we are using 'from' and 'sender' to mean the same thing in the same struct here.

Reply:
all right, this has been addressed everywhere

Reply:
Thanks! To be super duper nit-picky, it still slightly hurts my eyes to have `FromAccount` in one line and then `AccountFrom` in the next but perhaps that's just me :) I'll pipe down

Reply:
Yes, I'll make it AccountFrom, AccountTo, what do you think?

Reply:
That's fine by me but you'll obv. have to change it in all the places that currently use FromAccount. I did a quick grep of the codebase and FromAccount is mentioned 45 times and AccountFrom 108 so I think that's winning the popularity contest :) I really don't mind which as long as it's consistent!